### ISSUE TITLE
Find/Replace on multiple files copies contents of first file and replaces content of all subsequent files

### ISSUE THREAD

**Issue by mpresland** [microsoft/vscode#73011]
Issue Type: <b>Bug</b>

When doing a simple find/replace across multiple files, the string is being correctly found/replaced in the first file where the string is found. In subsequent files, the entire file is being replaced by the entire first file where the string is encountered.

Steps to reproduce: do a simple find and replace across multiple files.
![Screen Shot 2019-04-29 at 11 44 49 am](https://user-images.githubusercontent.com/16057824/56873268-857fb300-6a74-11e9-8a62-95c954c4aeef.png)

Result of first encountered file (expected result):
<img width="704" alt="Screen Shot 2019-04-29 at 11 45 35 am" src="https://user-images.githubusercontent.com/16057824/56873275-90d2de80-6a74-11e9-81a6-99fcf2bac49e.png">

Each subsequent file is overwritten by the first file:
<img width="704" alt="Screen Shot 2019-04-29 at 11 46 26 am" src="https://user-images.githubusercontent.com/16057824/56873281-a1835480-6a74-11e9-8fea-6d77abb9a5b2.png">

This issue has happened on 2 out of 3 find/replace actions this morning.

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:22:55.268Z)
OS version: Darwin x64 18.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (8 x 2600)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 2|
|Memory (System)|16.00GB (0.22GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (12)</summary>

Extension|Author (truncated)|Version
---|---|---
laravel-blade|cjh|1.1.2
vscode-eslint|dba|1.8.2
EditorConfig|Edi|0.13.0
php-intellisense|fel|2.3.10
todo-tree|Gru|0.0.130
php-cs-fixer|jun|0.1.118
php-constructor|Meh|0.1.2
php-namespace-resolver|Meh|1.1.7
php-docblocker|nei|1.8.0
vetur|oct|0.19.2
advanced-new-file|pat|1.2.0
vscode-fileutils|sle|2.13.7

</details>

**Comment by bpasero** [comment]
@mpresland a couple of questions:
* can you give us exact steps to reproduce, ideally with the workspace you did this in if open source?
* does it reproduce with our insiders build too? You can give our preview releases a try from: https://code.visualstudio.com/insiders/
* does it reproduce with other search/replace in other workspaces?

**Comment by bpasero** [comment]
@mpresland ping

**Comment by mpresland** [comment]
@bpasero 

 - it seems to happen when doing multiple find/replace actions one after the other. When we were refactoring some code, we needed to change the name of a few classes and interfaces across the project, and the issue never seemed to occur on the first find/replace we did. It always seemed to be on subsequent changes. unfortunately, the project is not available to share, it is a commercial project.
- I will try insiders build, no extensions etc when I get some time. I did initially suspect extensions, but I have yet to test extensively.
